modules/olm-channel-schema.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+[id="olm-channel-schema_{context}"]
+= olm.channel schema
+
+The `olm.channel` schema defines a channel within a package, the bundle entries that are members of the channel, and the upgrade edges for those bundles.
+
+A bundle can included as an entry in multiple `olm.channel` blobs, but it can have only one entry per channel.
+
+It is valid for an entry's replaces value to reference another bundle name that cannot be found in this catalog or another catalog. However, all other channel invariants must hold true, such as a channel not having multiple heads.
+
+.`olm.channel` schema
+[%collapsible]
+====
+[source,go]
+----
+#Channel: {
+  schema: "olm.channel"
+  package: string & !=""
+  name: string & !=""
+  entries: [...#ChannelEntry]
+}
+
+#ChannelEntry: {
+  // name is required. It is the name of an `olm.bundle` that
+  // is present in the channel.
+  name: string & !=""
+
+  // replaces is optional. It is the name of bundle that is replaced
+  // by this entry. It does not have to be present in the entry list.
+  replaces?: string & !=""
+
+  // skips is optional. It is a list of bundle names that are skipped by
+  // this entry. The skipped bundles do not have to be present in the
+  // entry list.
+  skips?: [...string & !=""]
+
+  // skipRange is optional. It is the semver range of bundle versions
+  // that are skipped by this entry.
+  skipRange?: string & !=""
+}
+----
+====
+
+[WARNING]
+====
+When using the `skipRange` field, the skipped Operator versions are pruned from the update graph and are therefore no longer installable by users with the `spec.startingCSV` property of `Subscription` objects.
+
+If you want to have direct (one version increment) updates to an Operator version from multiple previous versions, and also keep those previous versions available to users for installation, always use the `skipRange` field along with the `replaces` field. Ensure that the `replaces` field points to the immediate previous version of the Operator version in question.
```

Comment:
Note: remove parentheses

Suggested rewrite: 
You can update an Operator incrementally while keeping previously installed versions available to users for future installation by using both the 'skipRange' and 'replaces' field.